src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -105,7 +105,7 @@ public class CommandLineCmdletBase : PSCmdlet, IDisposable
         /// </summary>
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
Remoting loopback tests.Can execute command in a disconnected session
```powershell
Objects cannot be added to a closed buffer. Make sure the buffer is open for Add and Insert operations to succeed.
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersRemoting\HelpersRemoting.psm1: line 226
226:     Invoke-Command @parameters

```

Reply:
See comment [here](https://github.com/PowerShell/PowerShell/pull/9900#discussion_r415399412). I cannot reproduce this locally, but I've seen it turn up in a few other PRs if I'm remembering correctly, so I suspect it's another issue that is simply made more obvious by these changes.